### DISCUSSION TITLE
Shadn color scheme site

### DISCUSSION THREAD

**Discussion by mweel1** [shadcn-ui/ui#2461] [Ideas]
Would anyone be interested in a site that allowed people to upload Shadcn based stylesheets so users could browse different color schemes and then import them into their projects?

I'm finding myself having to build a bunch of color schemes and was thinking about building this if people were interested.

**Comment by samuelwhunter** [comment]
There's something similar here https://ui.jln.dev/?theme=RT8criw4NWtsMFzx. I've found it pretty useful

**Comment by cxa** [comment]
You can try this <https://realazy.com/enough-shadcn-ui-colors/> built by me. The advantage is its simplicity: you only need to select the desired hue, and it generates a more harmonious color scheme than other tools.

**Comment by wendynxinno** [comment]
Hi ! 😊
I’ve been working on[ josui.design](https://josui.design/), a small tool built with shadcn/ui and Tailwind.
It’s still a work in progress, but I’d be really happy if you’d give it a try and share any thoughts!

Thanks so much! 🙏
